spec/models/discipline_spec.rb

```diff
@@ -41,6 +41,16 @@
     end
   end
 
+  %i[name campus unity description category nature level].each do |field|
+    it "is invalid without #{field}" do
+      attrs = valid_att.clone
+      attrs.delete(field)
+
+      patent = described_class.new attrs
+      expect(patent).to be_invalid
+    end
+  end
+
   it 'is invalid without category' do
     attr = valid_att
     attr[:category][:innovation] = false
```

Comment:
Linhas `45` e `54` com o mesmo nome de teste quando o atributo for `category`, porém, testam coisas diferentes

Reply:
Mandei um novo commit alterando o nome do segundo teste, para deixar mais específico o que ele faz :)